components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -272,15 +284,6 @@ be lost if SCREAM_HACK_XML is not enabled.
     <!-- MAM4xx-Surface-Emissions -->
     <mam4_srf_online_emiss inherit="atm_proc_base">
 
-      <srf_emis_specifier_for_DMS    hgrid="ne4np4.pg2" type="file" doc="File containing surface emissions data for DMS">${DIN_LOC_ROOT}/atm/scream/mam4xx/emissions/ne4pg2/surface/DMSflux.2010.ne4pg2_conserv.POPmonthlyClimFromACES4BGC_c20240814.nc</srf_emis_specifier_for_DMS>
```

Comment:
This block is not removed. It is just moved below so that the xml parser can choose the right files for the specified resolution. The parser expects the files in a particular order.